### PR TITLE
Fix #5533 Software Architect Roadmap

### LINKED ISSUE #5533: Possible Misspelling
### What is the URL where the issue is happening

https://roadmap.sh/software-architect

### What browsers are you seeing the problem on?

Firefox

### Describe the Bug

I think the term "SAP EPR" should be "SAP ERP" unless the term "EPR" doesn't stand for Enterprise Resources Planning (ERP).   

erp-error

### Output from browser console (if any)

_No response_

### Participation

- [ ] I am willing to submit a pull request for this issue.

### PR DESCRIPTION
**Pull request by UtkarshTiwari1750**
Fix Issue: #5533 
Corrected misspelling
Before:- SAP EPR
After:- SAP ERP

### CODE REVIEW

File: package-lock.json
Comment (kamranahmedse) [inline package-lock.json:1]: Can you please remove `package-lock.json` changes from your PR.

File: src/data/roadmaps/software-architect/content/116-enterprise-software/101-sap-epr-hana-business-objects.md [hunk 1]
```diff
@@ -1 +1 @@
-# Sap epr hana business objects
+# Sap erp hana business objects
```
Comment (kamranahmedse) [inline src/data/roadmaps/software-architect/content/116-enterprise-software/101-sap-epr-hana-business-objects.md:1]:
```suggestion
# SAP ERP, HANA, Business Objects
```

**Review by kamranahmedse — DISMISSED** [review]

File: package-lock.json
Comment (UtkarshTiwari1750) [reply package-lock.json:1]: Ok. 
I have removed changes in package-lock.json and added your suggestion

**Review by dansholds — APPROVED** [review]
LGTM, thanks!

**Pull request merged**